### DISCUSSION TITLE
how to add new tmTheme files to use in massCode interface?

### DISCUSSION THREAD

**Discussion by thexmeta** [massCodeIO/massCode#421] [Q&A] [answered]
i found and fixed some of tmThemes for massCode but couldnt find how to implement in it to use it. Any idea how to make it?

**Comment by antonreshetov** [comment, marked as answer]
@thecodymetalist You can add a `tmTheme` [here](https://github.com/massCodeIO/massCode/tree/master/src/renderer/components/editor/themes), then add it to [here](https://github.com/massCodeIO/massCode/blob/master/src/renderer/components/editor/themes.ts). Since this only adds the theme to the editor, you need to take care of the rest of the UI by adding the necessary styles [here](https://github.com/massCodeIO/massCode/blob/master/src/renderer/assets/scss/themes.scss).
